include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -400,6 +400,13 @@
 #define MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_ONLY
 #endif
 
+/* psa_util file features some ECDSA conversion functions, to convert between
+ * legacy's ASN.1 DER format and PSA's raw one. */
+#if defined(MBEDTLS_ECDSA_C) || defined(MBEDTLS_PSA_CRYPTO_C) && \
```

Comment:
I can't remember precedence rules between `||` and `&&`, can you add parentheses to clarify? `ECDSA_C || (PSA_CRYPTO_C && (ALG_ECDSA || ALG_DET_ECDSA))`.